modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -6,10 +6,16 @@
 package com.liferay.source.formatter.check;
 
 import com.liferay.petra.string.CharPool;
+import com.liferay.petra.string.StringBundler;
```

Comment:
@TamyrisTorres, let's adjust the commit message to be like: `Create new method to add imports in JSP files`

Please see https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752578/Wordsmith+conventions+in+Liferay#%5BinlineExtension%5DHuman-readable-names%2C-not-Java-names

Reply:
resolved